omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py

```diff
@@ -2454,8 +2681,8 @@ def draw_scalebar_line(self, x, y, x2, y2, width, rgb):
         y2 = scale_to_export_dpi(y2)
         width = scale_to_export_dpi(width)
 
-        for value in range(-width // 2, width // 2):
-            draw.line([(x, y + value), (x2, y2 + value)], fill=rgb)
+        # the coordinates are included in the line
+        draw.line([(x, y), (x2, y2)], fill=rgb, width=width)
 
```

Comment:
That change was first to try to fix that pixel offset, but I realized it could be useful to not have independent lines object in the PDF. 
Note however that result would be different if x and x2 or y and y2 are swapped.
I will add code to ensure x2 < x and y2 < y (i think that's how it should be reproduce previous behavior)

Reply:
done